bioptim/optimization/optimal_control_program.py

```diff
@@ -712,52 +712,69 @@ def load(file_path: str) -> list:
             out = [ocp, sol]
         return out
 
+    def get_dynamics(self):
+        list_dynamics = []
+        for nlp in self.nlp:
+            list_dynamics.append(nlp.dynamics_type.type.name)
+        return list_dynamics
```

Comment:
I think it is okay to keep this function here (even though it could be moved in Dynamics or something). But this should not be a public method though. At least protected (_get_dynamics) or even private (__get_dynamics)

Reply:
I chose to define this method as a private method.

Reply:
Done